### PR TITLE
Change Margin Padding Helpers from horz and vert to `X` and `Y`

### PR DESCRIPTION
**Pull request by IamManchanda**
To keep with consistency with XY Grid!

=> The thing is now foundation community understands `X` and `Y`

### CODE REVIEW

**Comment by IamManchanda** [comment]
Added milestone for 6.5... but if possible in 6.4.x series then great too!

**Comment by kball** [comment]
Makes sense to me - lets do for 6.5 as it requires migration

**Comment by IamManchanda** [comment]
Ohk cool @kball lets merge it and then cherry pick to 6.5

File: docs/pages/prototyping-utilities.md [hunk 1]
```diff
@@ -554,6 +560,12 @@ Generate spaces around the content with these easy to use padding classes.
 <div class="padding-right-3"></div>
 ```
 
+<div class="callout warning">
+  <p>
+    <code>padding-horizontal</code> and <code>padding-vertical</code> classes have been depreciated and been replaced with <code>padding-x</code> and <code>padding-y</code> classes to keep our code consistent with <a href="xy-grid.html"><strong>XY Grid</strong></a>
```
Comment (kball) [inline docs/pages/prototyping-utilities.md]: Deprecated means they are still around but will be removed, lets change this simply to say they have been replaced.

File: docs/pages/prototyping-utilities.md [hunk 2]
```diff
@@ -461,22 +461,28 @@ Generate spacing around elements with these easy to use margin classes.
 <div class="margin-right-3"></div>
 ```
 
+<div class="callout warning">
+  <p>
+    <code>margin-horizontal</code> and <code>margin-vertical</code> classes have been depreciated and been replaced with <code>margin-x</code> and <code>margin-y</code> classes to keep our code consistent with <a href="xy-grid.html"><strong>XY Grid</strong></a>
```
Comment (kball) [inline docs/pages/prototyping-utilities.md]: Same note about deprecated here as below.

**Comment by IamManchanda** [comment]
Done @kball ... I think this is good to go now!

**Comment by IamManchanda** [comment]
Closing as per https://github.com/zurb/foundation-sites/pull/10625#issuecomment-327888834

**Pull request closed without merging**